### PR TITLE
fix: behavior planning launch

### PR DESCRIPTION
**Pull request by k-obitsu**
Signed-off-by: k-obitsu <koichi.obitsu@tier4.jp>

## PR Type

- Bug Fix

## Related Links

https://tier4.atlassian.net/browse/EVT4-1916
https://github.com/tier4/autoware_launch.x1.eva/pull/290
## Description

- Fixed the path of 'stop_line.param.yaml' in 'behavior_planning.launch.py'
- Related to the above, added  param.yaml files. 

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py [hunk 1]
```diff
@@ -167,6 +179,7 @@ def generate_launch_description():
             pull_over_param,
             pull_out_param,
             behavior_path_planner_param,
+            stop_line_param,
```
Comment (h-ohta) [inline planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py]: The stop line module is managed by behavior_velocity_planner.
So, you do not have to change this.

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py [hunk 2]
```diff
@@ -211,14 +224,6 @@ def generate_launch_description():
     with open(intersection_param_path, "r") as f:
         intersection_param = yaml.safe_load(f)["/**"]["ros__parameters"]
 
-    stop_line_param_path = os.path.join(
-        get_package_share_directory("behavior_velocity_planner"),
```
Comment (h-ohta) [inline planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py:215]: All you have to do is to change file path to followings.
```
        get_package_share_directory("planning_launch"),
        "config",
        "scenario_planning",
        "lane_driving",
        "behavior_planning",
        "behavior_velocity_planner",
        "stop_line.param.yaml",
```

File: planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py [hunk 3]
```diff
@@ -120,6 +120,18 @@ def generate_launch_description():
     with open(behavior_path_planner_param_path, "r") as f:
         behavior_path_planner_param = yaml.safe_load(f)["/**"]["ros__parameters"]
 
+    stop_line_param_path = os.path.join(
```
Comment (h-ohta) [inline planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py]: You do not need to add this series of change.
Refer to https://github.com/tier4/autoware_launch/pull/271/files#r845688430

File: planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/occlusion_spot.param.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/**:
+  ros__parameters:
+    occlusion_spot:
+      detection_method: "predicted_object"  # [-] candidate is "occupancy_grid" or "predicted_object"
+      pass_judge: "current_velocity"        # [-] candidate is "current_velocity"
```
Comment (taikitanaka3) [inline planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/occlusion_spot.param.yaml]: @k-obitsu 
can you sync this part please :pray: 
image

**Comment by takayuki5168** [comment]
LGTM except for Tanaka-san's comment.
@k-obitsu Please make sure that simple planning simulator works with this PR.

**Review by takayuki5168 — APPROVED** [review]
LGTM (I didn't verify planning simulation)

**Comment by k-obitsu** [comment]
I made sure the simple scenario with simple planning simulator.

**Comment by takayuki5168** [comment]
I confirmed that Tanaka-san's commet was resolved. Now it's ready to merge.

**Pull request merged**